wpeverest/user-registration
# UR-677 Fix - reCAPTCHA Registration Form templates don't create reCAPTCHA

### All Submissions:






### Changes proposed in this Pull Request:



Previously, the user can select  'Recaptcha Registration Form' template while adding a new form even though the captcha keys are not set. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Verify, whether the ''Recaptcha Registration Form' template is not being selected if captcha keys are not set.


### Types of changes:




### Other information:




### Changelog entry

> UR-677 Fix - reCAPTCHA Registration Form templates don't create reCAPTCHA.
